src/test/features/claim/routes/check-and-send.ts

```diff
@@ -58,6 +58,28 @@ describe('Claim issue: check and send page', () => {
           .set('Cookie', `${cookieName}=ABC`)
           .expect(res => expect(res).to.be.successful.withText('Check your answers'))
       })
+      it('Should validate the hyperlink (Change) available on checkAndSendPage  with correct location and span', async () => {
+        draftStoreServiceMock.resolveFind('claim')
+        feesServiceMock.resolveCalculateIssueFee()
```

Comment:
really minor - please add a line spacing here too, it will be very clear to spot mocks and rest of the test. we follow this every where...